src/migration/MigrationExecutor.ts

```diff
@@ -64,6 +64,13 @@ export class MigrationExecutor {
     public async executeMigration(migration: Migration): Promise<Migration> {
         return this.withQueryRunner(async (queryRunner) => {
             await this.createMigrationsTableIfNotExist(queryRunner)
+
+            // create typeorm_metadata table if it's not created yet
+            const schemaBuilder = this.connection.driver.createSchemaBuilder()
+            if (InstanceChecker.isRdbmsSchemaBuilder(schemaBuilder)) {
+                await schemaBuilder.createTypeormMetadataTable(queryRunner)
```

Comment:
replace `createTypeormMetadataTable()` with `createMetadataTableIfNecessary()` and check if it resolves your issue.